src/witan/models/dem/ccm/components_functions.clj

```diff
@@ -78,3 +68,80 @@
                                vec
                                (sort-by (apply juxt col-key))
                                ds/dataset)))
+
+(defn lag
```

Comment:
very nice! Please could you explain to me how this works? I would love to understand